src/components/Navigation/index.js

```diff
@@ -34,6 +49,11 @@ export default class SiderDemo extends React.Component<
   };
 
   render() {
+    if (this.state.reload) {
+      this.setState({ reload: false });
+      return <Redirect to="/" />;
```

Comment:
Which component does RR4 choose to mount in this case? I’m genuinely curious about this. 🤔

Reply:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Redirect.md#to-string

This will pass through the router and route the user to the onboarding page if the localStorage value is not set